package org.openprovenance.prov.model.exception;

/**
 * An exception thrown when a documented unsupported case is called
 *
 * @author lavm
 *
 */
public class DocumentedUnsupportedCaseException extends RuntimeException {


    /**
     *
     */
    public DocumentedUnsupportedCaseException() {
        // TODO Auto-generated constructor stub
    }

    /**
     * @param message a message
     */
    public DocumentedUnsupportedCaseException(String message) {
        super(message);
        // TODO Auto-generated constructor stub
    }

    /**
     * @param cause the exception
     */
    public DocumentedUnsupportedCaseException(Throwable cause) {
        super(cause);
        // TODO Auto-generated constructor stub
    }

    /**
     * @param message a message
     * @param cause the exception
     */
    public DocumentedUnsupportedCaseException(String message, Throwable cause) {
        super(message, cause);
        // TODO Auto-generated constructor stub
    }


}
